llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -3337,8 +3337,13 @@ static bool isLegalArithImmed(uint64_t C) {
   return IsLegal;
 }
 
-static bool cannotBeIntMin(SDValue CheckedVal, SelectionDAG &DAG) {
-  KnownBits KnownSrc = DAG.computeKnownBits(CheckedVal);
+static bool cannotBeIntMin(SDValue Op, SelectionDAG &DAG) {
```

Comment:
This function needs to be renamed since it no longer receives the operand and now receives the operation.